src/main/java/com/tasks/taskapi/repository/TaskRepository.java

```diff
@@ -38,4 +43,10 @@ public int insertTask(Task task) {
 
         return (int) keyHolder.getKey();
     }
-}
+
+    public void editTask(Task task) {
+        String EDITED_EDIT_SQL = String.format(EDIT_SQL + WHERE_SQL, task.getName(), 
```

Comment:
formattedEditSql ? also would only use upper snake case at the top of the class but just convention